src/ontology/edits/oeo-physical.omn

```diff
@@ -11867,6 +11867,24 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1486",
     SubClassOf: 
         OEO_00320072,
         OEO_00020056 some OEO_00000159
+
+Class: OEO_00320061
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Vehicle-kilometre is a transport performance unit for the accumulated transport distance of the used vehicles themselves where one vehicle-kilometre equals the transport distance of 1 km for one vehicle.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1375
+pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1388
+
+restructure modules:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
+
+ <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "vkm",
+        rdfs:comment "Can be obtained from passenger-kilometre/ton-kilometre by dividing through the average number of passengers/tons per vehicle respectively.",
+        rdfs:label "vehicle-kilometre"
+    
+    SubClassOf: 
+        OEO_00320001
```

Comment:
This line probably causes the error. OEO_00320001 is used here, but not declared. If you add a line 
```
Class: OEO_00320001
```
it should work